### PR TITLE
use %d in format string for discovery port

### PR DESCRIPTION
**Pull request by ryane**
most of this is just from go fmt

### CODE REVIEW

File: mesos/register.go [hunk 1]
```diff
@@ -129,16 +129,16 @@ func (m *Mesos) registerTask(t *state.Task, agent string) {
...
 		servicePort := strconv.Itoa(discoveryPort.Number)
 		log.Debugf("%+v framework has %+v as a name for %+v port",
 			t.Name,
 			discoveryPort.Name,
 			discoveryPort.Number)
-		if discoveryPort.Name != ""{
+		if discoveryPort.Name != "" {
 			m.Registry.Register(&registry.Service{
-				ID:      fmt.Sprintf("mesos-consul:%s:%s:%s", agent, tname, discoveryPort.Number),
+				ID:      fmt.Sprintf("mesos-consul:%s:%s:%d", agent, tname, discoveryPort.Number),
```
Comment (ryane) [inline mesos/register.go]: changing %s to %d here is the only significant change

File: mesos/register.go [hunk 2]
```diff
@@ -2,10 +2,10 @@ package mesos
 
 import (
 	"fmt"
-	"strings"
-	"strconv"
 	"github.com/CiscoCloud/mesos-consul/registry"
 	"github.com/CiscoCloud/mesos-consul/state"
+	"strconv"
```
Comment (ChrisAubuchon) [inline mesos/register.go]: Why change the order of imports?

Comment (ryane) [inline mesos/register.go]: afaik, that is just something gofmt does.

Comment (ChrisAubuchon) [inline mesos/register.go]: Ok. It's the lack of a blank line between `"strconv"` and `github.com/.../registry`.

**Comment by ChrisAubuchon** [comment]
I've got a different PR that doesn't change the order of the imports. The `%d` fix is there.

**Pull request closed without merging**